src/mapping/nav2_map_server/test/unit/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+include_directories( ${PROJECT_SOURCE_DIR}/test )
+
+ament_add_gtest(test_occ_grid test_occ_grid.cpp ${PROJECT_SOURCE_DIR}/test/test_constants.cpp)
+
+ament_target_dependencies(test_occ_grid rclcpp nav_msgs)
+
+target_link_libraries(test_occ_grid
+  map_reps
+  stdc++fs
+  ${ament_LIBRARIES}
```

Comment:
These ament_* macros don't seem to work if you try to call make directly, which seems to be desirable for debugging. I think it is better to spell out all the dependencies explicitly ( though it could be in a variable you define if you need to specify the values repeatedly)